prow/oss/create-build-cluster.sh

```diff
@@ -28,6 +28,7 @@ export CLUSTER="${CLUSTER:-${PROJECT}}"
 export GCS_BUCKET="${GCS_BUCKET:-gs://${PROJECT}}"
 
 # Only needed for creating cluster
+export GITHUB_FORK_URI="${GITHUB_FORK_URI:-}"  # Set this to the URI of your personal fork of this repo.
```

Comment:
Thank you! Might be helpful to mention it looks very similar to line 21

Reply:
Done.
/hold cancel